components/compositing/compositor_thread.rs

```diff
@@ -22,40 +22,54 @@ use style_traits::viewport::ViewportConstraints;
 use webrender;
 use webrender_traits;
 
-/// Sends messages to the compositor. This is a trait supplied by the port because the method used
-/// to communicate with the compositor may have to kick OS event loops awake, communicate cross-
-/// process, and so forth.
-pub trait CompositorProxy : 'static + Send {
-    /// Sends a message to the compositor.
-    fn send(&self, msg: Msg);
-    /// Clones the compositor proxy.
-    fn clone_compositor_proxy(&self) -> Box<CompositorProxy + 'static + Send>;
+
+/// Used to wake up the event loop, provided by the servo port/embedder.
+pub trait EventLoopRiser : 'static + Send {
```

Comment:
How about `EventLoopAwaker`, so that we have a noun? https://en.wiktionary.org/wiki/awaker